packages/terra-collapsible-menu-view/src/CollapsibleMenuViewDivider.jsx

```diff
@@ -2,22 +2,45 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import Menu from 'terra-menu';
+import IconChevronRight from 'terra-icon/lib/icon/IconChevronRight';
 import ThemeContext from 'terra-theme-context';
 import styles from './CollapsibleMenuView.module.scss';
 
 const cx = classNames.bind(styles);
 
+const propTypes = {
+  /**
+   * @private
+   * Uses a ">" icon instead of a line.
+   */
+  useChevronRightIcon: PropTypes.bool,
+};
+
 const contextTypes = {
   isCollapsibleMenuItem: PropTypes.bool,
 };
 
-const CollapsibleMenuViewDivider = (props, { isCollapsibleMenuItem }) => {
+const CollapsibleMenuViewDivider = ({ useChevronRightIcon }, { isCollapsibleMenuItem }) => {
   const theme = React.useContext(ThemeContext);
+  if (useChevronRightIcon) {
+    if (isCollapsibleMenuItem) {
```

Comment:
I understand why this is here, but without the knowledge that the right chevron divider is made for the Filepath component, this is maybe unintuitive that it affects how the menu looks. A clarifying comment here might help. I'm also wondering if naming the prop differently (ex: isFilepath) might make it clear it changes more than just the divider styling while also describing its intended use case.

Reply:
I can add a comment saying that it will not be rendered in the collapsed menu view! However, I'm not sure about naming it `isFilepath` because all these changes try to not couple collapsible menu view and filepath as much as possible.